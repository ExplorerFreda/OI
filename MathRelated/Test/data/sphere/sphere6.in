8
15471.000000 14970.000000 2461.500000 4376.000000 5121.000000 7429.000000 19477.500000 317.500000
3253.000000 6899.000000 5339.000000 2724.000000 3878.500000 5998.000000 16369.500000 15116.500000
4871.500000 7064.500000 17569.000000 4514.500000 9130.500000 6665.000000 4929.500000 1171.500000
3269.500000 4795.500000 2532.000000 9662.000000 3226.500000 19650.500000 6538.000000 13515.000000
14536.500000 2075.000000 15633.000000 4824.000000 15516.000000 10547.500000 5078.500000 19753.500000
19648.000000 18882.500000 1724.000000 3197.500000 4657.500000 1827.000000 18173.000000 19012.000000
2678.000000 4575.500000 11928.500000 4469.000000 19675.500000 15481.000000 3746.000000 1532.000000
1585.000000 12511.000000 13760.000000 15355.500000 13006.000000 7588.000000 9415.500000 1406.000000
17174.000000 741.500000 5851.500000 10831.500000 3664.000000 4442.000000 9461.000000 6796.500000
