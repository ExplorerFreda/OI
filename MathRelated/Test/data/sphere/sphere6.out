11675.967 16781.974 1261.549 5098.278 13905.226 7311.516 1339.217 10866.925
